### ISSUE TITLE
'Advanced' Expand/Collapse control is not accessible under 'New collection' blade.

### ISSUE THREAD

**Issue by ato9000** [Azure/cosmos-explorer#1013]
Repro Steps:
Open URL: https://ms.portal.azure.com/ and login using valid credentials.
Navigate to "Azure Cosmos DB " from left pane and select it.
Navigate to any of the Azure Cosmos DB which is already created and select it.
Navigate to "Add Collection" Menu item/Button and select it.
Navigate through all the controls under add Collection/New collection blade.
Navigate to 'Shard' key edit field using tab key.
Press the tab key now to move to next interactive control i.e. 'Advanced' Expand/Collapse control.
Verify the issue.

Actual Result:
'Advanced' Expand/Collapse control is not accessible under 'New collection' blade.

Expected Result:
'Advanced' Expand/Collapse control which is not accessible using mouse should also be accessible through keyboard under 'New collection' blade.

Note:
'Tabstop' is not working on this page and blades.
This issue also exists for similar control present under 'New Graph' blade of Data explorer screen for Created cosmos DB  after selecting "Gremlin(graph)" option.

https://user-images.githubusercontent.com/8519407/132721913-16f1f91d-82c9-45c8-962d-4fb07b92cc7b.mp4

**Comment by ato9000** [comment]
https://msdata.visualstudio.com/CosmosDB/_workitems/edit/1216093
